SoftLayer/CLI/image/export.py

```diff
@@ -12,17 +12,23 @@
 @click.command()
 @click.argument('identifier')
 @click.argument('uri')
+@click.option('--ibm-api-key',
+              default="",
```

Comment:
Consider changing default to None, rather than empty string.

Reply:
Will add on next commit.